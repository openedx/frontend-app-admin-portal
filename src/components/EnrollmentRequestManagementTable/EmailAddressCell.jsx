import React from 'react';
import PropTypes from 'prop-types';

const EmailAddressCell = ({ row }) => (
  <span data-hj-suppress>{row.original.emailAddress}</span>
);

EmailAddressCell.propTypes = {
  row: PropTypes.shape({
    original: PropTypes.shape({
      emailAddress: PropTypes.string,
    }),
  }).isRequired,
};

export default EmailAddressCell;
