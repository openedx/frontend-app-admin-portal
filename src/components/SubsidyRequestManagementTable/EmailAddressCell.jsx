import React from 'react';
import PropTypes from 'prop-types';

const EmailAddressCell = ({ row }) => (
  <span data-hj-suppress>{row.original.email}</span>
);

EmailAddressCell.propTypes = {
  row: PropTypes.shape({
    original: PropTypes.shape({
      email: PropTypes.string,
    }),
  }).isRequired,
};

export default EmailAddressCell;
