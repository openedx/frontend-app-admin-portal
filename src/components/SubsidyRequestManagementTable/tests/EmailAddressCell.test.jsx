import React from 'react';
import renderer from 'react-test-renderer';

import EmailAddressCell from '../EmailAddressCell';

const defaultProps = {
  row: {
    original: {
      email: 'test@example.com',
    },
  },
};

describe('EmailAddressCell', () => {
  test('renders as expected', () => {
    const tree = renderer
      .create(<EmailAddressCell {...defaultProps} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
