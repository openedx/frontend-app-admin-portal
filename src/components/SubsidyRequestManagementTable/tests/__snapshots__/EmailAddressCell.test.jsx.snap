// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailAddressCell renders as expected 1`] = `
<span
  data-hj-suppress={true}
>
  test@example.com
</span>
`;
